### PR TITLE
Fetch featured stories

### PR DESCRIPTION
**Pull request by ostigley**

### CODE REVIEW

File: lib/supplejack/story.rb [hunk 1]
```diff
@@ -139,6 +139,17 @@ def self.find(id, user_key: nil, params: {})
       end
     end
 
+    # Fetches featured stories
+    # Oliver Stigley July 2017
+    #
+    def self.featured
+      begin
+        get('/stories/featured')
```
Comment (Reinkaos) [inline lib/supplejack/story.rb]: Can /stories/featured return an empty array rather than raising an error?

File: lib/supplejack/story.rb [hunk 2]
```diff
@@ -139,6 +139,17 @@ def self.find(id, user_key: nil, params: {})
...
+      rescue RestClient::ResourceNotFound
+        raise Supplejack::StoryNotFound, "Features stories not found"
```
Comment (Reinkaos) [inline lib/supplejack/story.rb]: 'Featured stories not found'

File: lib/supplejack/story.rb (hunk 1 above)
Comment (ostigley) [inline lib/supplejack/story.rb]: @hapiben Do you agree?  I'm happy to take it off.  The view will still render with an empty array

Comment (ostigley) [inline lib/supplejack/story.rb]: @Reinkaos I've removed this

**Comment by hapiben** [comment]
👍

**Comment by Reinkaos** [comment]
👍

**Pull request merged**